src/Npgsql/NpgsqlReadBuffer.Stream.cs

```diff
@@ -180,6 +218,19 @@ protected override void Dispose(bool disposing)
                     _buf.Skip(leftToSkip, false).GetAwaiter().GetResult();
                 IsDisposed = true;
             }
+
+#if !NET461 && !NETSTANDARD2_0
+            public override async ValueTask DisposeAsync()
```

Comment:
Makes sense to change `Disposed` to call its async version.

Reply:
Done.